webrtc.html

```diff
@@ -5001,6 +5007,29 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
         </section>
       </div>
       <div>
+        <pre class="idl">dictionary RTCRtpCodecRtxParameters {
+             unsigned short payloadType;
+             unsigned long rtxTime;
+};</pre>
+        <section>
+          <h2>Dictionary <a class="idlType">RTCRtpCodecRtxParameters</a> Members</h2>
+          <dl data-link-for="RTCRtpCodecRtxParameters" data-dfn-for=
+          "RTCRtpCodecRtxParameters" class="dictionary-members">
+            <dt><dfn><code>payloadType</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned short</a></span></dt>
+            <dd>
+              <p>The retransmission payload type.</p>
+            </dd>
+            <dt><dfn><code>rtxTime</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long</a></span></dt>
+            <dd>
+              <p>The maximum time (measured in milliseconds) a sender will keep
+              an original RTP packet in its buffers available for retransmission.</p>
```

Comment:
Neither me. But AFAIU `RtpParameters` are currently generated by the browser based on the given SDP offer/answer, so such a `rtxTime` may be read-only (not sure if the spec allows that within `RtpParameters`).